### ISSUE TITLE
ERC20 DataType Error

### ISSUE THREAD

**Issue by codesport** [0xSplits/splits-contracts#1]
Hi Guys,

Love your project. Hoping you can help.

**TLDR:** How can we get ethers on the frontend to recognize the ERC20 dataType when calling [`getERC20Balance()`](https://github.com/0xSplits/splits-contracts/blob/7d3e225e2826a3649a63336456b932cebd1ac0a8/contracts/SplitMain.sol#L580)

---
We're building a dApp to interact with your splitter on Polygon. 

However, on the frontend, ethers.js is throwing the following error when we connect with your `getERC20Balance` function. The error is:

`Uncaught (in promise) Error: invalid type (argument="type", value="ERC20", code=INVALID_ARGUMENT, version=abi/5.6.4)`

Specifically it doesn't like the `ERC20` dataType in [line 580](https://github.com/0xSplits/splits-contracts/blob/7d3e225e2826a3649a63336456b932cebd1ac0a8/contracts/SplitMain.sol#L580) of `SplitsMain.sol`.   When I remove the call to this function everything works fine.

We were able to clone your repo locally and all test passed. 

Any, guidance, how to get ethers not to complain about this ERC20 dataType?

**Comment by wminshew** [comment]
hey -- the ERC20 data type is only used by the solidity compiler. The ABI & typechain outputs (https://github.com/0xSplits/splits-sdk, also available via npm) simply take addresses as strings

if you're still having issues, @mihoward21 may be able to help more next week

**Comment by codesport** [comment]
Hi @wminshew 

Thanks for tip!

We got it working!! 🎉🎈
